step-templates/f5-ltm-rest-enable-disable-member.json

```diff
@@ -0,0 +1,114 @@
+  },
+  "Parameters": [
+    {
+      "Id": "5ed914ec-ff22-436d-ac37-1ccc5216372e",
+      "Name": "WaitForConnections",
+      "Label": "Wait on Connections?",
+      "HelpText": "If checked, the deployment won't continue until current connections on the node = 0",
+      "DefaultValue": "False",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      }
+    },
+    {
+      "Id": "2207ca51-ce9c-4498-8394-e0553fbcf54c",
+      "Name": "MaxWaitTime",
+      "Label": "Max. Wait Time (seconds)",
+      "HelpText": "Defaults to 5 minutes",
+      "DefaultValue": "300",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "7f503c0b-0050-4685-a9d8-7dbc42199ac0",
+      "Name": "EnableOrDisable",
+      "Label": "LTM Status",
+      "HelpText": "State member will be put into (Enabled, Disabled, Offline)  \n\n- **Disabled** continues to process persistent and active connections. It can accept new connections only if the connections belong to an existing persistence session.  \n\n- **Offline** allows existing connections to time out, but no new connections are allowed.",
+      "DefaultValue": "Disabled",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Select",
+        "Octopus.SelectOptions": "Enabled|Enable\nDisabled|Disable\nOffline|Forced Offline"
+      }
+    },
+    {
+      "Id": "e49bb08a-c72b-4bae-95ea-bb2c4c808657",
+      "Name": "PoolName",
+      "Label": "LTM Pool name",
+      "HelpText": "Name of F5 pool",
+      "DefaultValue": "pl_DummyDeployWeb",
```

Comment:
I don't think you need a default value here.

Reply:
removed default value from pool name in commit d0d386fa4e5f44f76445fb09d530ea4e94a96fc2